src/hooks/use-authentication/index.ts

```diff
@@ -84,16 +77,12 @@ const useAuthentication = (
 
 	// Return everything packaged up in an object
 	return {
-		error,
 		isAuthenticated,
 		isLoading,
 		session,
-		showAuthenticatedUI,
-		showUnauthenticatedUI,
 		signIn,
 		signOut,
 		signUp,
-		status,
```

Comment:
All the ones removed here were either not used or could be inferred from data already passed in other properties. In an effort to simplify this hook, I decided to remove them!